docs/bots-guide.md

```diff
@@ -346,6 +346,62 @@ client.update_message(dict(
 * My bot works only on some streams.
     * Subscribe your bot to other streams, as described [here](#how-to-run-a-bot).
 
+## Flask Server
```

Comment:
Instead of putting this section at the very end, how would you feel about putting it right underneath the "How to run a bot" section?